medical_appointment/data/medical_appointment_sequence.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-TODAY
+	     LasLabs [https://laslabs.com]
+	     Komit [komit-consulting.com]
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
-<!--
-    Copyright (C) 2015-TODAY LasLabs [https://laslabs.com]
-    @author Dave Lasley <dave@laslabs.com>
-    @license AGPL-3
--->
-
-<openerp>
+<odoo>
```

Comment:
I wanted to refer to this line:
`The tag <data> is only used to set not-updatable data with noupdate=1`

I can submit the PR if you like, but it seems that you are already deeply engaged on the topic (https://github.com/OCA/maintainer-tools/pull/253) :)